cmd/crictl/logs.go

```diff
@@ -75,6 +81,10 @@ var logsCommand = &cli.Command{
 			Aliases: []string{"t"},
 			Usage:   "Show timestamps",
 		},
+		&cli.StringFlag{
+			Name:  "stream",
```

Comment:
Do we want to add an alias?

```suggestion
			Name:  "stream",
			Aliases: []string{"s"},
```

Reply:
I think it is great to add one.